packages/release-mastermind/src/action.ts

```diff
@@ -12,314 +1,547 @@
+	opts: Options
+	configJSON: Options["configJSON"]
+	configPath: Options["configPath"]
+	dryRun: Options["dryRun"]
+	fillEmpty: Options["fillEmpty"]
+	repo = context.repo || {}
+	util: Utils
+
+	constructor(client: GitHub, options: Options) {
+		log(
+			LoggingLevels.debug,
+			`Release Mastermind Constructed: ${options.toString()}`
+		)
+		core.startGroup("Setup Phase")
+		this.client = client
+		this.opts = options
+		this.dryRun = options.dryRun
+		if (this.dryRun) {
+			if (options.repo) this.repo = options.repo
+			if (!options.repo?.repo)
+				this.repo.repo = process.env.GITHUB_REPOSITORY || "Unknown"
+			if (!options.repo?.owner)
+				this.repo.owner = process.env.GITHUB_REPOSITORY_OWNER || "Unknown"
+		}
+		this.configJSON = options.configJSON
+		this.configPath = options.configPath
+		this.fillEmpty = options.fillEmpty
+		this.util = new Utils(
+			{ client, repo: this.repo },
+			{ dryRun: options.dryRun, skipDelete: options.skipDelete }
+		)
+	}
 
-  constructor(client: GitHub, options: Options) {
-    log(
-      
-        LoggingLevels.debug,
-        `Release Mastermind Constructed: ${options.toString()}`
-    )
-    core.startGroup('Setup Phase')
-    this.client = client
-    this.opts = options
-    this.configJSON = options.configJSON
-    this.configPath = options.configPath
-    this.util = new Utils({ client, repo: this.repo }, { dryRun: options.dryRun, skipDelete: options.skipDelete })
-    this.dryRun = options.dryRun
-    this.fillEmpty = options.fillEmpty
-  }
+	async run() {
+		log(LoggingLevels.debug, `Repo data: ${this.repo.owner}/${this.repo.repo}`)
 
-  async run() {
-    if (this.dryRun) this.repo.repo = process.env.GITHUB_REPOSITORY || 'Unknown'
-    if (this.dryRun)
-      this.repo.owner = process.env.GITHUB_REPOSITORY_OWNER || 'Unknown'
-    log(
-      LoggingLevels.debug, `Repo data: ${this.repo.owner}/${this.repo.repo}`
-    )
+		/**
+		 * Capture and log context to debug for Local Running
+		 * @author TGTGamer
+		 * @since 1.0.0
+		 */
+		log(
+			LoggingLevels.debug,
+			`Context for local running. See readme.md for information on how to setup local running: ${JSON.stringify(
+				context
+			)}`
+		)
+		/**
+		 * Process the config
+		 * @author TGTGamer
+		 * @since 1.1.0
+		 */
+		log(LoggingLevels.debug, `Config: ${JSON.stringify(this.configJSON)}`)
 
-    /**
-     * Capture and log context to debug for Local Running
-     * @author TGTGamer
-     * @since 1.0.0
-     */
-    log(
-      
-        LoggingLevels.debug,
-        `Context for local running. See readme.md for information on how to setup local running: ${JSON.stringify(
-          context
-        )}`
-      )
-    /**
-     * Process the config
-     * @author TGTGamer
-     * @since 1.1.0
-     */
-    log(LoggingLevels.debug, `Config: ${JSON.stringify(this.configJSON)}`)
+		const configs = await this.processConfig().catch((err) => {
+			throw log(
+				LoggingLevels.error,
+				`Error thrown while processing config: `,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
        "Error thrown while processing config: ",
```